sqlite-db/src/load_rollover_completed_event_data.rs

```diff
@@ -0,0 +1,369 @@
+use crate::Sqlite;
+use anyhow::Result;
+use bdk::bitcoin::hashes::hex::FromHex;
+use bdk::bitcoin::secp256k1;
+use bdk::bitcoin::Address;
+use bdk::bitcoin::Amount;
+use bdk::bitcoin::Script;
+use bdk::descriptor::Descriptor;
+use model::Cet;
+use model::Dlc;
+use model::FundingFee;
+use model::OrderId;
+use model::RevokedCommit;
+use sqlx::Transaction;
+use std::collections::HashMap;
+use std::ops::RangeInclusive;
+use std::str::FromStr;
+
+/// Load RolloverCompleted event data
+///
+/// Returns Ok(Some(..) if one was found or Ok(None) if none was found.
+/// In error case, it returns Err(..)
+pub async fn load_rollover_completed_event_data(
+    transaction: &mut Transaction<'_, Sqlite>,
+    offer_id: OrderId,
+    event_row_id: i64,
+) -> Result<Option<(Dlc, FundingFee)>> {
+    let revoked_commit = load_revoked_commit_transactions(transaction, offer_id).await?;
+    let cets = load_cets(transaction, offer_id).await?;
+
+    let row = sqlx::query!(
+            r#"
+            SELECT
+                main_data.settlement_event_id as "settlement_event_id: model::olivia::BitMexPriceEventId",
```

Comment:
🙃 I would find it more intuitive to use the table name here, i.e. `rollover_completed_event_data` instead of giving it an alias `main_data` that is not expressive.

Reply:
Will do!

Reply:
[`84afd77` (#2140)](https://github.com/itchysats/itchysats/pull/2140/commits/84afd77b10e004d0aa0f9f8afb7c68f4907102b9)